Engine/Component/ComponentLight.h

```diff
@@ -28,8 +28,8 @@ class ComponentLight : public Component
 			void ChangePointLightAttenuationValues(float range)
 			{
 				float augmented_range = 10 * range;
-				linear = 4.5 / (10 * augmented_range);
-				quadratic = 75.0 / (augmented_range*augmented_range);
+				linear = 4.5f / (10 * augmented_range);
+				quadratic = 75.0f / (augmented_range*augmented_range);
```

Comment:
macros self explanatory to know why this values?

Reply:
I think I commented this on another PR, this values are defined [here](http://wiki.ogre3d.org/Light+Attenuation+Shortcut). I dont really know the reason of why they are these concrete values.